### DISCUSSION TITLE
`docx` template documentation

### DISCUSSION THREAD

**Discussion by estedeahora** [jgm/pandoc#10273] [Q&A] [answered]
I am investigating some of the recent changes in the Word writer. From what I understand a template was added that allows (together with ‘reference-doc’) to customise the order of elements in a docx output (commit [db559e1](https://github.com/jgm/pandoc/commit/db559e100c02ca1f95953f3eeeca005fdc01b595)). However, I believe this was not documented in the [manual](https://github.com/jgm/pandoc/blob/de034b70f8fce864f9d83fa4c61b83f61c87cbf0/MANUAL.txt#L2250C1-L2251C1). Should this be added or am I misunderstanding something?

**Comment by jgm** [comment, marked as answer]
Yes, it should be documented somehow.  At the very least, under `--template` we should note that an `openxml` template will affect the docx writer and an `opendocument` template will affect the odt writer.  Perhaps that's enough.  For further customizations, just as with any format, it's best to start by looking at the default template -- and then you'd need to consult OpenXML documentation.

**Reply by estedeahora** [reply]
I tried to commit this (#10274). I don't know if I succeeded because I did it online.
